### ISSUE TITLE
change train loader in demo.py

### ISSUE THREAD

**Issue by bkvie** [ChunhuanLin/deform_conv_pytorch#2]
train_loader = torch.utils.data.DataLoader(
    datasets.MNIST(**'/home/chlin/data/dataset/MNIST**', train=True, download=True,
                   transform=transforms.Compose([
                       transforms.ToTensor(),
                       transforms.Normalize((0.1307,), (0.3081,))
                   ])),
    batch_size=args.batch_size, shuffle=True, **kwargs)

change the hard coded file path to **./**

**Comment by forrestl111** [comment]
I have updated the code. Thanks for your advice!
